pages/linux/systemd-detect-virt.md

```diff
@@ -0,0 +1,24 @@
+# systemd-detect-virt
+
+> Detect execution in a virtualized environment.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-detect-virt.html>.
+
+- List detectable virtualization technologies:
+
+`systemd-detect-virt --list`
+
+- Detect virtualization, print the result and return a 0 status code when running in a VM or a container, and a non-zero code otherwise:
```

Comment:
```suggestion
- Detect virtualization, print the result and return a zero status code when running in a VM or a container, and a non-zero code otherwise:
```
Minor nitpick